.gitignore

```diff
@@ -115,6 +115,9 @@ venv.bak/
 dj_local_conf.json
 root/.one_params
 
+# docker related file
+docker-compose.yml
```

Comment:
did we want to do the same `-template` rename for the 'local' .yml file as well?